tests/foreman/ui/test_oscapcontent.py

```diff
@@ -64,9 +68,13 @@ def test_positive_end_to_end(session, oscap_content_path):
         assert org.name in oscap_values['organizations']['resources']['assigned']
         assert loc.name in oscap_values['locations']['resources']['assigned']
         session.oscapcontent.update(title, {'file_upload.title': new_title})
+        session.location.search('abc')
```

Comment:
Why is this hard coded

Reply:
@san7ket I have added it as workaround for issue [https://github.com/SatelliteQE/airgun/issues/382](https://github.com/SatelliteQE/airgun/issues/382).

Reply:
Please add a comment here , pointing to the issue

Reply:
@san7ket updated the pr.